Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -286,9 +288,16 @@ class SourceGenerator {
             let isTopLevelGroup = (isBaseGroup && !createIntermediateGroups) || isRootPath
 
             let groupName = name ?? path.lastComponent
-            let groupPath = isTopLevelGroup ?
-                ((try? path.relativePath(from: project.basePath)) ?? path).string :
```

Comment:
The previous implementation seems to be wrong.

According to the `PBXGroup`'s documentation, `path` argument must be relative path from source root.
https://github.com/tuist/XcodeProj/blob/aefcbf59cea5b0831837ed2f385e6dfd80d82cc9/Sources/XcodeProj/Objects/Files/PBXGroup.swift#L28

However, in some situation, `groupPath` become `path.string`. It is absolute path.